### PR TITLE
Expose function test

### PR DESCRIPTION
**Pull request by parostatkiem**
**Description**
The test was based on [this](https://kyma-project.io/docs/main/root/getting-started#getting-started-expose-the-function) guide. The _in-memory-cache_ part was skipped on purpose.
Suggested changes for the future: 
Create a namespace once per all test suites, create a function afterwards. Perform all other tests and leave the _expose the function_ part for the end. This will save up a lot of time waiting for the function to be ready.

Changes proposed in this pull request:

- Improve namespace deletion a little bit (still not perfect).
- get the Function code dynamically from the Guide 
- create an API Rule for the function 
- check if the generated `host` is valid
- make a request to the `host` and check the response

**Related issue(s)**

### CODE REVIEW

File: resources/web/deployment.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ spec:
     spec:
       containers:
         - name: busola
-          image: eu.gcr.io/kyma-project/busola-core:PR-144
+          image: eu.gcr.io/kyma-project/busola-core:PR-136
```
Comment (Sawthis) [inline resources/web/deployment.yaml:19]:
```suggestion
          image: eu.gcr.io/kyma-project/busola-core:PR-144
```

File: tests/tests/busola-create-a-deployment.spec.js [hunk 2]
```diff
@@ -84,9 +82,21 @@ context('Busola - Create a Deployment', () => {
   });
 
   it('Go to the details of namespace', () => {
+    // console.log(cy.url());
```
Comment (Sawthis) [inline tests/tests/busola-create-a-deployment.spec.js:85]: do we need this commented code?

**Review by Sawthis — CHANGES_REQUESTED** [review]

**Review by Sawthis — APPROVED** [review]

**Review by akucharska — APPROVED** [review]

**Review by klaudiagrz — APPROVED** [review]

**Pull request merged**